docs/01-module-block-1/00-structure.adoc

```diff
@@ -2,19 +2,11 @@
 // (c) iSAQB e.V. (https://isaqb.org)
 // ====================================================
 
-// tag::DE[]
-== Lerneinheit 1
-// end::DE[]
-
-// tag::EN[]
-== Lesson 1
-// end::EN[]
+== {lesson} 1
```

Comment:
Nitpick: "Lerneinheit 1" is bad German. Better would be "1. Lerneinheit".

Reply:
Cannot be changed easily at the moment. :(